test_naucse/test_markdown.py

```diff
@@ -91,6 +91,38 @@ def test_markdown_definition_list_advanced():
     assert convert_markdown(src).strip() == expected
 
 
+def test_markdown_ansi_colors():
+    src = dedent("""
+        ```ansi
+        On branch ansicolors
+        Changes to be committed:
+          (use "git reset HEAD <file>..." to unstage)
+
+            [32mmodified:   naucse/markdown_util.py[m
+
+        Changes not staged for commit:
+          (use "git add <file>..." to update what will be committed)
+          (use "git checkout -- <file>..." to discard changes in working directory)
+
+            \x1b[31mmodified:   test_naucse/test_markdown.py\x1b[m
```

Comment:
I'm worried about putting control characters in the Markdown file.
I would prefer replacing it by another character. Then people could at least find it in their editor.
I guess  `␛` (U+241B SYMBOL FOR ESCAPE) would be perfect for this.